db/setup.sql

```diff
@@ -83,7 +83,8 @@ VALUES
 ('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A22', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A12', 'amzn', 'nasdaq', 100000000),
 ('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A23', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A13', 'fb', 'nasdaq', 100000000),
 ('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A24', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A14', 'msft', 'nasdaq', 100000000),
-('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A25', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A15', 'brk.a', 'nyse', 100000000);
+('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A25', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A15', 'brk.a', 'nyse', 100000000),
+('A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A26', 'A0EEBC99-9C0B-4EF8-BB6D-6BB9BD380A15', 'nflx', 'nasdaq', 500);
```

Comment:
Added a second holding to user "gramps" in db setup so you can see multiple stock holdings right away in the friend search

Reply:
👍